syntax = "proto3";

package api.chat.request;

message AddMember {
    uint64 chat_id = 1;
    uint64 user_id = 2;
}
